### PR TITLE
operationName and variables fields added to graphql json parsing

### PR DESCRIPTION
**Pull request by rikysya**

### CODE REVIEW

File: pkg/satellite/satelliteweb/api.go [hunk 1]
```diff
@@ -28,6 +28,12 @@ const (
 	applicationGraphql = "application/graphql"
 )
 
+type queryJSON struct {
```
Comment (crawter) [inline pkg/satellite/satelliteweb/api.go]: it will be better to have some comments here

File: pkg/satellite/satelliteweb/api.go [hunk 2]
```diff
@@ -68,32 +76,30 @@ func getToken(req *http.Request) string {
...
 
 // queryPOST retrieves query from POST request
-func queryPOST(req *http.Request) (query string, err error) {
+func queryPOST(req *http.Request, query *queryJSON) (err error) {
 	switch typ := req.Header.Get(contentType); typ {
 	case applicationGraphql:
 		body, err := ioutil.ReadAll(req.Body)
-		return string(body), utils.CombineErrors(err, req.Body.Close())
+		query.Query = string(body)
+		return utils.CombineErrors(err, req.Body.Close())
```
Comment (crawter) [inline pkg/satellite/satelliteweb/api.go]: return utils.CombineErrors(err, req.Body.Close())

this line could be moved from applicationGraphql and applicationJSON cases and placed after the switch

File: pkg/satellite/satelliteweb/api.go [hunk 3]
```diff
@@ -68,32 +77,30 @@ func getToken(req *http.Request) string {
...
-func getQuery(req *http.Request) (query string, err error) {
+func getQuery(req *http.Request) (query graphqlJSON, err error) {
 	switch req.Method {
 	case http.MethodGet:
-		return req.URL.Query().Get(satelliteql.Query), nil
+		query.Query = req.URL.Query().Get(satelliteql.Query)
+		return query, nil
 	case http.MethodPost:
-		return queryPOST(req)
+		return query, queryPOST(req, &query)
```
Comment (egonelbre) [inline pkg/satellite/satelliteweb/api.go]: From `queryPost` return `query` and the error, then you can do here `return queryPost(req)`

File: pkg/satellite/satelliteweb/api.go [hunk 4]
```diff
@@ -28,6 +28,13 @@ const (
 	applicationGraphql = "application/graphql"
 )
 
+// JSON response from graphql clients
```
Comment (egonelbre) [inline pkg/satellite/satelliteweb/api.go]: Isn't this the request?

**Pull request merged**